src/components/author/author.test.ts

```diff
@@ -27,11 +27,11 @@ test(
 
     const body = await page.evaluateHandle<ElementHandle>(() => document.body);
     expect(await getAccessibilityTree(body)).toMatchInlineSnapshot(`
-        text "By"
-        link "Shakira Isabel Mebarak Ripoll"
-          text "Shakira Isabel Mebarak Ripoll"
-        text "Presented on March 31st, 2021"
-      `);
+      text "By"
```

Comment:
eslint-plugin-unicorn now has a rule to auto-fix jest's dumb inline snapshot indentation!!